index.js

```diff
@@ -1,19 +1,19 @@
 'use strict';
 const flags = require('flags');
 const fs = require('fs');
-const tmp = require('tmp');
 
-const extractScriptsFromHTML = require('./src/extractScripts.js');
+// The order of operation is to extract each js script from HTML, transform it,
+// then inject it back into HTML.
+const {extractScriptsFromHTML} = require('./src/extractScripts.js');
+const {transformSourceCodeString} = require('./src/transform.js');
+const {injectScriptsIntoHTML} = require('./src/injectScripts.js');
 
 const FILE_FLAG = 'file';
 const DIR_FLAG = 'dir';
-const KEEP_TEMP_FLAG = 'keepTemp';
 
 // Specify exactly one of --file or --dir.
 flags.defineString(FILE_FLAG, null, 'Path to test file to transform');
 flags.defineString(DIR_FLAG, null, 'Path to dir of test files to transform');
-flags.defineBoolean(KEEP_TEMP_FLAG, false,
-    'If true, keep the temporary directories of extracted scripts upon exit.');
```

Comment:
Optional: It'd be nice to provide debug flags to introspect e.g. extracted scripts. This can be a follow-up.

Reply:
It will be a follow up! Need to think a bit more on how I want that organized.